components/Choice/index.jsx

```diff
@@ -83,6 +85,8 @@ export default class Choice extends Component {
       requiredLabel
     } = this.props;
 
+    const optionsPerRow = this.props.optionsPerRow && parseInt(this.props.optionsPerRow, 10);
```

Comment:
Hm, I think the problem is, that we probably still want to support showing all options in one row, don't we? If I had a default here, The options would always break into multiple rows. 

However if we want this, I think it would probably be a good idea since we wouldn't have data-options-per-row={null} in the components (this doesn't get rendered to html, but still)